vars/shared-OpenData.var.yml

```diff
@@ -85,6 +85,30 @@ extensions:
       url: "https://github.com/ckan/ckanext-harvest.git"
       version: "v1.2.0"
 
+    CKANExtArchiver: &CKANExtArchiver
+      name: "ckanext-archiver-{{ Environment }}"
+      shortname: "ckanext-archiver"
+      description: "CKAN Extension for Archiving needed for ckanext-qa"
+      type: "git"
+      url: "https://github.com/qld-gov-au/ckanext-archiver.git"
+      version: "master"
```

Comment:
This should use a specific tag or commit if it's in environments higher than DEV.